test cases/common/140 get define/concat.h

```diff
@@ -0,0 +1,23 @@
+#define __STRINGIFY(x) #x
+#define TEST_STRINGIFY(x) __STRINGIFY(x)
+
+#define TEST_VERSION_MAJOR 6
+#define TEST_VERSION_MINOR 0
+#define TEST_VERSION_BUGFIX 0
+
+#define TEST_VERSION_STR                                       \
+    TEST_STRINGIFY(TEST_VERSION_MAJOR)                         \
+    "." TEST_STRINGIFY(TEST_VERSION_MINOR) "." TEST_STRINGIFY( \
+        TEST_VERSION_BUGFIX)
+
+#define TEST_CONCAT_1 \
+    "ab"              \
+    "cd"              \
+    "ef"
```

Comment:
These are some nice tests. Could you also add a `""` at the end of this to test that it gets removed?